utils/state/state.test.ts

```diff
@@ -0,0 +1,11 @@
+import { LoadingState } from '@/constants';
+
+import { isLoading } from './index';
+const makeSut = () => ({ sut: isLoading });
+describe('state', () => {
+  it('Should be a true if loading state is updating or fetching', () => {
+    const { sut } = makeSut();
+    const result = sut(LoadingState.Idle);
```

Comment:
set up a file 

set up testing environment when u test components:

use the files below as instructions and the repo:

https://github.com/ilovemochi/mochi-website/tree/develop
https://github.com/ilovemochi/mochi-website/blob/develop/modules/test-utils.tsx
https://github.com/ilovemochi/mochi-website/blob/develop/.jest/setup.ts
https://github.com/ilovemochi/mochi-website/blob/develop/jest.config.js

Reply:
Test all possible cases and nuances